src/components/Form/FormProvider.js

```diff
@@ -73,6 +73,8 @@ const propTypes = {
 
     /** Should validate function be called when the value of the input is changed */
     shouldValidateOnChange: PropTypes.bool,
+
+    shouldHideFixErrorsAlert: PropTypes.bool,
```

Comment:
Every other prop has comment above, maybe we should add short description to `shouldHideFixErrorsAlert`